plugins/rssurlrewrite/init.js

```diff
@@ -1,42 +1,40 @@
 plugin.loadMainCSS();
-if(browser.isIE && browser.versionMajor < 8)
+if (browser.isIE && browser.versionMajor < 8)
 	plugin.loadCSS("ie");
 plugin.loadLang();
 
-theWebUI.curRule = null;
-theWebUI.maxRuleNo = 0;
+plugin.curRule = null;
+plugin.maxRuleNo = 0;
+plugin.rules = [];
```

Comment:
These attributes should be local to the plugin itself, instead of the global `theWebUI` object.